content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+{{< glossary_definition term_id="eviction" length="all" >}}
+
+During an eviction, the kubelet sets the `PodPhase` for the selected pods to `Failed`.
+This gracefully terminates the pods. 
+
+If the pods are managed by a workload resource (such as {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}
+or {{< glossary_tooltip text="Deployment" term_id="deployment" >}}) that
+replaces failed pods, the control plane creates new pods in place of the evicted pods.
+
+{{<note>}}
+The kubelet attempts to [reclaim node-level resources](#reclaim-node-resources)
+before it terminates end-user pods.
+{{</note>}}
+
+When evicting pods, the kubelet respects your configured [PodDisruptionBudget](/docs/tasks/run-application/configure-pdb/).
+You can control which pods are evicted last by setting the [Quality of Service class](/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed)
```

Comment:
This is incorrect.  QoS has no impact on eviction ordering.  See https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/#evicting-end-user-pods for what is taken into account.

Reply:
Thanks for the comments! 

The [Priority and Preemption page](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#interactions-of-pod-priority-and-qos) says that the only time Priority and QoS are both considered is during out-of-resource eviction. 

Also, the out of resource handling section has `BestEffort` and `Burstable` going before `Guaranteed`. Isn't that considering the QoS?

Reply:
Yeah, we have been discussing that recently, as you aren't the only one confused by it.  We don't actually take QoS into account, but since QoS is based on setting requests, and requests relative to usage, you can reason from QoS to rough ordering of evictions.  For example, a Guaranteed pod is still going to be evicted last simply because it is impossible for its usage to exceed requests, which is the first criterion we rank by.  Suggestions would be welcome, but we have been discussing removing all references to QoS here.

Reply:
We should update the Priority and Preemption page.  I wasn't aware of that section.  Note that the text after mentioning QoS where it talks about the algorithm doesn't mention QoS at all...  It is just the headline that is wrong there.

Reply:
@dashpole please feel very welcome to file an issue about any incorrect content you see!

Reply:
I created #27753 and added a PR for it. Accidentally didn't make the PR a draft, sorry.